src/components/molecules/VenuePartygoers/VenuePartygoers.scss

```diff
@@ -0,0 +1,3 @@
+.attendees-container {
```

Comment:
Should this be scoped under something `VenuePartygoers` related?

Generally in the past I've used patterns where the top 'container' uses a 'wrapper' class (eg. `.venue-partygoers`), and then all other more specific rules live under this. That way they will only be applied when they appear within the 'wrapping classes scope'.

That way we can be more confident that our CSS styling doesn't weirdly change/break other things, without having to result to a CSS-in-JS pattern (which in many cases feels heavy/overkill IMO)

Reply:
We will benefit tremendously from a documented convention for wrapper classes around components. Something like:

* A component must render a top-level element with a `className` of `COMPONENTNAME-wrapper`
* The .scss must be entirely wrapped in
```
.COMPONENTNAME-wrapper {
...
}
```

Reply:
Agreed. Potentially something we can lint (`stylelint`?), if not with an existing rule, with a custom. But until then, documented/enforced convention sounds good to me.

Reply:
@thecadams RE: the above, I have some good patterns I figured out with how we can use CSS modules (enabled by default in `create-react-app` + some other improvements to our styling. Started documenting my context in https://github.com/sparkletown/internal-sparkle-issues/issues/4 the other day to avoid the rabbithole, but the main relevant part to this would be:

- https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

You can use generic names like `.error` and similar in the component's css module, then import it like `import styles from './Button.module.css'`, which then allows you to use it like `<button className={styles.error}>Error Button</button>;`

It generates locally scoped unique classnames, so no global clashes, and isn't as heavy as a lot of CSS-in-JS patterns/libs.